src/transform.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+const b = require('@babel/core');
```

Comment:
nit: call it babel instead of b? 4 more chars in a lot of places but i think it will read nicer

Reply:
Will do. I might refactor later to directly import `template` and the other used libraries independently but for now that's a good suggestion.